apps/meteor/app/lib/server/functions/setUserActiveStatus.ts

```diff
@@ -52,12 +53,30 @@ export async function setUserActiveStatus(userId: string, active: boolean, confi
 		return false;
 	}
 
+	if (user.active === active) {
+		return true;
+	}
```

Comment:
Do we really to change this here?

Reply:
if I don't do it here, one level up in the meteor method I'd have to (more ideal) but then we'd be querying the db for the same user twice. or change function signature

Reply:
If I don't check, if someone hits the endpoint for an already active user, it will still show the error. let me see if I can move it around

Reply:
@MarcosSpessatto pls check :)